docker-compose.yml

```diff
@@ -13,10 +13,9 @@ services:
     environment:
       F8_AUTH_URL: "http://localhost:8089"
       F8_DEVELOPER_MODE_ENABLED: "true"
-      F8_POSTGRES_HOST: db
-      F8_POSTGRES_PORT: 5432
     ports:
       - "8080:8080"
+    network_mode: "host"
     depends_on:
       - auth
   db-auth:
```

Comment:
You missed network_mode: "host" here? And for WIT db too.

Reply:
maybe host is used by default. Just checked it and the PR works fine on my Linux.

Reply:
Honestly, I restored the file as it was in https://github.com/fabric8-services/fabric8-wit/blob/18b9c1f3da21df9d55e279db5f79afcc25a4d1ad/docker-compose.yml (changed 8 days ago)